packages/config/config/devices/0x0344/he-hls01.json

```
@@ -0,0 +1,455 @@
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Click function disabled",
+					"value": 0
+				},
+				{
+					"label": "Toggle switch",
+					"value": 1
+				},
+				{
+					"label": "Relay switches to ON state only",
+					"value": 2
+				},
+				{
+					"label": "Relay switches to OFF state only",
+					"value": 3
+				},
+				{
+					"label": "Timer On>Off mode",
+					"value": 4
+				},
+				{
+					"label": "Time Off>On mode",
+					"value": 5
+				}
+			]
+		},
+		"71": {
+			"label": "Relay Timer Mode Duration",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0
+		},
+		"141": {
+			"label": "Energy Consumption Meter Consecutive Report Interval",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 120,
+			"defaultValue": 10
+		},
+		"142": {
+			"label": "Energy Consumption Report Threshold",
+			"description": "Consumption difference to send to controller, %",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 50,
+			"defaultValue": 25
+		},
+		"143": {
+			"label": "Temperature sensor consecutive report interval",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 120,
+			"defaultValue": 10
+		},
+		"144": {
+			"label": "Temperature report threshold",
+			"description": "Temperature difference to send to controller, value X 10",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 2
+		},
+		"151": {
+			"label": "Device Failure Notifications",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"152": {
+			"label": "Temperature Rise Notification",
+			"description": ": We recommend setting the value of this Parameter lower than value of Parameter 153 (Overheat Protection).",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 120,
+			"defaultValue": 1
+		},
+		"153": {
+			"label": "Overheat Protection & Notification",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 120,
+			"defaultValue": 0
+		},
+		"154": {
+			"label": "Over-Current Notification",
+			"description": "Even if you turn off the notification, the device will switch Off the load when the current reaches 16A.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"155": {
+			"label": "Over-Load Protection & Notification",
+			"description": "The value of this parameter can be set from 100 to 4000 in watts. Use the value 0 if there is a need to disable this function.",
```

```suggestion
			"description": "Allowable Range 100-4000.",
```
and add `Disable` as an option.